tests/legacy/unit-tests/rest-api/Tests/Version3/products.php

```diff
@@ -218,7 +219,7 @@ public function test_update_product() {
 		$product      = \Automattic\WooCommerce\RestApi\UnitTests\Helpers\ProductHelper::create_simple_product();
 		$response     = $this->server->dispatch( new WP_REST_Request( 'GET', '/wc/v3/products/' . $product->get_id() ) );
 		$data         = $response->get_data();
-		$date_created = date( 'Y-m-d\TH:i:s', current_time( 'timestamp' ) );
+		$date_created = gmdate( 'Y-m-d\TH:i:s', current_time( 'timestamp' ) );
```

Comment:
Should we add a test for `null` and `""` just in case we forget about those again?

Reply:
agree, added in https://github.com/woocommerce/woocommerce/pull/27318/commits/dd9b9cfae2a3f565dddc90d135486c9cf1e58ba1